.taskcluster.yml

```diff
@@ -1,125 +1,264 @@
-              type: file
-              path: /opt/firefox-tv/parameters.yml
-              expires: {$fromNow: '1 year'}
-        extra:
-          tasks_for: ${tasks_for} # for chain of trust to re-build the task
-        metadata:
-          name: 'Decision task'
-          description: ''
-          source: ${repository}/raw/${head_rev}/.taskcluster.yml
-    in:
-      - $if: 'tasks_for == "github-pull-request" && event["action"] in ["opened", "reopened", "edited", "synchronized"]'
-        then:
-          $mergeDeep:
-            - {$eval: 'default_task_definition'}
-            - scopes:
-                - ${assume_scope_prefix}:pull-request
-              payload:
-                command:
-                  - >-
-                    git fetch ${repository} ${event.pull_request.head.ref}
-                    && git config advice.detachedHead false
-                    && git checkout FETCH_HEAD
-                    && python tools/taskcluster/decision_task.py pull-request ${event.sender.login}@users.noreply.github.com ${event.pull_request.head.ref}
-              metadata:
-                owner: ${event.sender.login}@users.noreply.github.com
-      - $if: 'tasks_for == "github-push" && event["ref"] == "refs/heads/master"'
-        then:
-          $mergeDeep:
-            - {$eval: 'default_task_definition'}
-            - scopes:
-                - ${assume_scope_prefix}:branch:master
-              payload:
-                command:
-                  - >-
-                    git fetch ${repository} refs/heads/master
-                    && git config advice.detachedHead false
-                    && git checkout FETCH_HEAD
-                    && python tools/taskcluster/decision_task.py master ${event.sender.login}@users.noreply.github.com
-              metadata:
-                owner: ${event.sender.login}@users.noreply.github.com
-      - $if: 'tasks_for == "github-release" && event["action"] == "published"'
-        then:
-          $mergeDeep:
-            - {$eval: 'default_task_definition'}
-            - scopes:
-                - ${assume_scope_prefix}:release
-              payload:
-                command:
-                  - >-
-                    git fetch ${repository} refs/tags/${event.release.tag_name}
-                    && git config advice.detachedHead false
-                    && git checkout FETCH_HEAD
-                    && python tools/taskcluster/decision_task.py release ${event.release.tag_name}
-              metadata:
-                owner: firefox-tv@mozilla.com
+    - $let:
+          taskgraph:
+              branch: taskgraph
+              revision: 0c5a68749f9a7672a7e56604b69a7bd41b036614
+          trustDomain: mobile
+      in:
+          $let:
+              # Github events have this stuff in different places...
+              ownerEmail:
+                  $if: 'tasks_for == "github-push"'
+                  then: '${event.pusher.email}'
+                  # Assume Pull Request
```

Comment:
nit: this seems inaccurate because there are both PRs and releases below.

Reply:
Good eye